src/curated-corpus/helpers/definitions.ts

```diff
@@ -73,3 +73,13 @@ export type ApprovedItemFromProspect = Omit<
   language: CorpusLanguage | undefined;
   authors: CollectionStoryAuthor[] | CorpusItemAuthor[];
 };
+
+// List of scheduled surfaces used to show the readable name by mapping it to it's corresponding guid
+export const ScheduledSurfaces = [
+  { guid: 'NEW_TAB_EN_US', name: 'New Tab (en-US)' },
+  { guid: 'NEW_TAB_DE_DE', name: 'New Tab (de-DE)' },
+  { guid: 'NEW_TAB_EN_GB', name: 'New Tab (en-GB)' },
+  { guid: 'NEW_TAB_EN_INTL', name: 'New Tab (en-INTL)' },
+  { guid: 'POCKET_HITS_EN_US', name: 'Pocket Hits (en-US)' },
+  { guid: 'POCKET_HITS_DE_DE', name: 'Pocket Hits (de-DE)' },
+];
```

Comment:
I wonder if there's a better way to approach this, even if it takes a bit more time. We managed to get the hardcoded values out of the frontend repo entirely with the addition of the [`getScheduledSurfacesForUser`](https://github.com/Pocket/curated-corpus-api/blob/cefb9fdca562775add3bb37b6c6d3af9c3def117/schema-admin.graphql#L868) query on the backend. 

This returns a list of `ScheduledSurface` objects that contain both the GUID and the name of the scheduled surface, so that it's super easy to generate dropdown options, for example, such as [this bit of code over here](https://github.com/Pocket/curation-admin-tools/blob/cdfbd8c035dfb0711b1696815607a1d35e3a54ab/src/curated-corpus/pages/ProspectingPage/ProspectingPage.tsx#L108). 

What if we had another backend query that returned _all_ surfaces, as I imagine we'll need this in more than one place, for example for the upcoming page for each curated item with its own history? 🤔 That would ensure the list of scheduled surfaces is hardcoded in one place only - on the backend.

Reply:
cc @jpetto for their input on this :).

Reply:
i'm having trouble landing confidently on a suggestion, but i think it's this:

for now, hard code the data in this repo, but copy/paste it from our [shared data doc](https://getpocket.atlassian.net/wiki/spaces/PE/pages/2584150049/Pocket+Shared+Data#Source-of-Truth.5).

here's my rationale:

- our shared data is meant to be implemented in a copy/paste manner at this time, and it's already happening in prospect-api and curated-corpus-api, as well as at least one ML repo.
- the long(er)-term plan is to have our shared data available as a service (from _some_ api), which would make a query added to curated-corpus-api unnecessary.
- _if_ we do make a query in curated-corpus-api, someone else is going to use it before long, and then we'll have another client we need to migrate to the actual shared data api when it exists. basically, i am afraid curated-corpus-api would accidentally become the canonical way to retrieve that specific piece of shared data - and maybe it should be! but that decision needs more strategic planning around it.
- we often have to update this repo's generated types when surface shared data is updated anyway - meaning a PR is likely already required for surface updates.